install_config/http_proxies.adoc

```diff
@@ -93,8 +94,36 @@ host name as a suffix, or the host name itself.
 
 For instance, *example.com* would match *example.com*, *example.com:80*, and *www.example.com*.
 
-[[proxying-docker-pull]]
+[[configuring-hosts-for-proxies]]
+== Configuring Hosts for Proxies using Ansible
+
+During
+xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[advanced installations], 
+the `*NO_PROXY*`, `*HTTP_PROXY*`, and `*HTTPS_PROXY*` environment variables can
+be configured using the
+xref:../install_config/install/advanced_install.html#advanced-install-configuring-global-proxy[`*openshift_no_proxy*`, `*openshift_http_proxy*`, and `*openshift_https_proxy*` parameters], 
+which are configurable in the inventory file.
+
+.Example Proxy Configuration with Ansible
+====
 
+----
+# Global Proxy Configuration
+# These options configure HTTP_PROXY, HTTPS_PROXY, and NOPROXY environment
+# variables for docker and master services.
+openshift_http_proxy=http://USER:PASSWORD@IPADDR:PORT
+openshift_https_proxy=https://USER:PASSWORD@IPADDR:PORT
+openshift_no_proxy='.hosts.example.com,some-host.com'
+#
+# Most environments don't require a proxy between openshift masters, nodes, and
+# etcd hosts. So automatically add those hostnames to the openshift_no_proxy list.
+# If all of your hosts share a common domain you may wish to disable this and
+# specify that domain above.
+#openshift_generate_no_proxy_hosts=True
+----
+====
+
```

Comment:
We likely want to add a note about the other proxy settings that can also be configured, as [referanced](https://docs.openshift.com/enterprise/3.2/install_config/install/advanced_install.html#advanced-install-configuring-global-proxy), and point people to the following sections where proxies are used for builds (and how the ansible settings can help you set variables like this globally). 

- https://docs.openshift.com/enterprise/3.2/dev_guide/builds.html#using-a-proxy-for-git-cloning
- https://docs.openshift.com/enterprise/3.2/dev_guide/builds.html#docker-strategy-environment
- https://docs.openshift.com/enterprise/3.2/dev_guide/builds.html#custom-strategy-environment